backend/src/services/incrementalSyncService.js

```
@@ -0,0 +1,513 @@
+        } catch (error) {
+            console.log(`[SYNC] Could not update sync state: ${error.message}`);
+        }
+    }
+
+    /**
+     * Check if holder list has changed significantly
+     */
+    static async checkHolderListChanges() {
+        console.log('[SYNC] Checking for holder list changes...');
+        
+        const holderSyncState = await this.getSyncState('holder_list');
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)